src/firebase-import.js

```diff
@@ -80,29 +82,74 @@ function promptToContinue(ref, next) {
   }
 }
 
+function getJsonFromFile(file, callback) {
+  var firstChar;
+  var rs = fs.createReadStream(file);
+  rs.on('data', function(chunk) {
+    var s = chunk.toString().trim();
+    if (s !== "") {
+      rs.close();
```

Comment:
I was confused by this code until I realized your goal was just to read the first character.  Maybe factor this out into its own getFirstChar(file, callback) function?  (or better yet, return a Promise if you feel like it)